app/assets/stylesheets/components/helpers/_variables.scss

```diff
@@ -10,8 +10,8 @@ $app-spacing-scale-6: 40px;
 $app-spacing-scale-7: 50px;
 $app-spacing-scale-8: 60px;
 
-$app-text-colour: $text-colour;
-$app-secondary-text-colour: $grey-1;
+$app-text-colour: $govuk-text-colour;
+$app-secondary-text-colour: govuk-colour("dark-grey", $legacy: "grey-1");
 
 // Border widths
 $app-border-width-mobile: 4px;
```

Comment:
Line 24: `$app-error-colour: $red;` - does "$red" come from toolkit? I think we can remove this line and update where `$app-error-colour` is used (as mentioned in a previous comment)

Reply:
Done, good spot, thanks.